Source/Charts/Charts/ChartViewBase.swift

```diff
@@ -475,41 +447,28 @@ open class ChartViewBase: NSUIView, ChartDataProvider, AnimatorDelegate
     /// Highlights the value selected by touch gesture.
     @objc open func highlightValue(_ highlight: Highlight?, callDelegate: Bool)
     {
-        var entry: ChartDataEntry?
-        var h = highlight
-        
-        if h == nil
-        {
-            _indicesToHighlight.removeAll(keepingCapacity: false)
-        }
-        else
-        {
-            // set the indices to highlight
-            entry = _data?.entryForHighlight(h!)
-            if entry == nil
-            {
-                h = nil
+        var high = highlight
+        guard
+            let h = high,
```

Comment:
can we use single let here

Reply:
Unfortunately, no. I tried, but couldn't find a way because we need to assign the original highlight.